library/bind.c

```diff
@@ -265,6 +261,17 @@ static uint8_t get_terminator_precedence(const ziti_listen_opts *opts, const cha
 static int dispose(ziti_connection server) {
     assert(server->type == Server);
 
+    model_map_iter it = model_map_iterator(&server->server.bindings);
+    while(it) {
+        struct binding_s *b = model_map_it_value(it);
+        if (!b->bound && b->waiter == NULL) {
+            it = model_map_it_remove(it);
+            free(b);
```

Comment:
don't need to free b->bound or b->waiter, right? i assume not but always better to ask i suppose :)

Reply:
waiters are owned by the channel